python/lsst/afw/cameraGeom/utils.py

```diff
@@ -759,29 +915,52 @@ def showCamera(camera, imageSource=FakeImageDataSource(), imageFactory=afwImage.
                detectorNameList=None, binSize=10, bufferSize=10, frame=None, overlay=True, title="",
                showWcs=None, ctype=afwDisplay.GREEN, textSize=1.25, originAtCenter=True, display=None,
                **kwargs):
-    """!Show a Camera on display, with the specified display
+    """Show a Camera on display, with the specified display
 
     The rotation of the sensors is snapped to the nearest multiple of 90 deg.
-    Also note that the pixel size is constant over the image array. The lower left corner (LLC) of each
-    sensor amp is snapped to the LLC of the pixel containing the LLC of the image.
-    if overlay show the IDs and detector boundaries
-
-    @param[in] camera  Camera to show
-    @param[in] imageSource  Source to get Ccd images from.  Must have a getCcdImage method.
-    @param[in] imageFactory  Type of image to make
-    @param[in] detectorNameList  List of names of Detectors to use. If None use all
-    @param[in] binSize  bin factor
-    @param[in] bufferSize  size of border in binned pixels to make around camera image.
-    @param[in] frame  specify image display (@deprecated; new code should use display)
-    @param[in] overlay  Overlay Detector IDs and boundaries?
-    @param[in] title  Title in display
-    @param[in] showWcs whether to include a WCS in the display
-    @param[in] ctype  Color to use when drawing Detector boundaries
-    @param[in] textSize  Size of detector labels
-    @param[in] originAtCenter Put origin of the camera WCS at the center of the image? Else it will be LL
-    @param[in] display  image display on which to display
-    @param[in] **kwargs all remaining keyword arguments are passed to makeImageFromCamera
-    @return the mosaic image
+    Also note that the pixel size is constant over the image array. The lower
+    left corner (LLC) of each sensor amp is snapped to the LLC of the pixel
+    containing the LLC of the image.
+
+    Parameters
+    ----------
+    camera : `lsst.afw.cameraGeom.Camera`
+        Camera object to use to make the image.
+    imageSource : `class` with getCcdImage method
+        Source to get ccd images.  Must have a `getCcdImage()` method.
+    imageFactory : `lsst.afw.image.Image`
+        Type of image to make
+    detectorNameList : `list`
+        List of detector names from `camera` to use in building the image.
+        Use all Detectors if None.
+    binSize : `int`
+        Step size between samples.
+        i.e. `np.linspace(0, bbox.getHeight(), bbox.getHeight()/binSize)`
```

Comment:
The "i.e." ("that is") is saying how it's used, not what it is.

Reply:
Where do we stand here?  Paul's saying that this is e.g. not i.e. (and I agree), but I don't think that this is a helpful doc string even then.  The original doc for `binSize` in `makeImageFromCcd` was "Bin the image by this factor in both dimensions" which is implemented in `ccdImage = afwMath.binImage(ccdImage, binSize)` and I think that that's what we should be saying here.

Reply:
I'll go with that one.

There were at least 4 different definitions of the `binSize` parameter used in the various functions defined in this file. It would have helped if they were all self-consistent to start with.